app/meson.build

```diff
@@ -16,6 +16,7 @@ src = [
     'src/server.c',
     'src/strutil.c',
     'src/tinyxpm.c',
+    'src/apkinstaller.c'
```

Comment:
(move it to preserve alphabetical order – except `main.c`)
(maybe I should also sort `main.c` with the others)